agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/DiseaseAnnotationCurationIndexer.java

```diff
@@ -256,6 +256,13 @@ private static int getPhylogeneticSortOrder(String taxonID) {
 		return phylogeneticSortOrder;
 	}
 
+	private void populateImplicatedDA(DiseaseAnnotation original, DiseaseAnnotation implicatedDA) {
```

Comment:
Not sure why this code is added here as nothing int he DiseaseAnnotationCurationIndexer calls it.